shm/Store.cpp

```diff
@@ -1,40 +1,40 @@
 #include "Store.hpp"
 
-Response Store::buy(Cargo* cargo, size_t amount, Player* player){
-    if(!cargo){
+Response Store::buy(Cargo* cargo, size_t amount, Player* player) {
+    if (!cargo) {
         return Response::lack_of_cargo;
     }
-    if(!player){
+    if (!player) {
         return Response::lack_of_cargo;
     }
-    if(!amount){
+    if (!amount) {
         return Response::lack_of_cargo;
```

Comment:
I think that you can catch these 3 "if conditions" to 1
and do the same in sell method

Reply:
Done.